### ISSUE TITLE
micrometer-registry-atlas dependency added in v0.5.2.6

### ISSUE THREAD

**Issue by dannpopescu** [confluentinc/parallel-consumer#625]
Hi, in the latest 0.5.2.6 version the `micrometer-registry-atlas` dependency was added along with `micrometer-core`. I'm wondering if this is intentional or if it was supposed to be used only in the testing phase.

It is strange that a library makes a choice about a specific registry (this is something that should be left to the application I think). We noticed this because Spring Boot auto-configures the Atlas exporter and there are a lot of warning logs mentioning that it failed to send metrics.

```
failed to send metrics (uri=http://localhost:7101/api/v1/publish)
java.net.ConnectException: Connection refused
	at sun.nio.ch.Net.pollConnect(Net.java)
	at sun.nio.ch.Net.pollConnectNow(Net.java:672)
	at sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:542)
	at sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:597)
	at java.net.Socket.connect(Socket.java:633)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:178)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:531)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:636)
	at sun.net.www.http.HttpClient.<init>(HttpClient.java:279)
	at sun.net.www.http.HttpClient.New(HttpClient.java:384)
	at sun.net.www.http.HttpClient.New(HttpClient.java:406)
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1309)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1242)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1128)
	at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:1057)
	at sun.net.www.protocol.http.HttpURLConnection.getOutputStream0(HttpURLConnection.java:1430)
	at sun.net.www.protocol.http.HttpURLConnection.getOutputStream(HttpURLConnection.java:1401)
	at com.netflix.spectator.ipc.http.HttpRequestBuilder.sendImpl(HttpRequestBuilder.java:371)
	at com.netflix.spectator.ipc.http.HttpRequestBuilder.send(HttpRequestBuilder.java:288)
	at com.netflix.spectator.atlas.impl.DefaultPublisher.lambda$publish$0(DefaultPublisher.java:168)
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.lang.Thread.run(Thread.java:833)
```

**Comment by JorgenRingen** [comment]
Same issue here: https://github.com/confluentinc/parallel-consumer/issues/618

**Comment by rkolesnev** [comment]
Fixed by #628
